services/workers/src/main/java/org/sagebionetworks/report/worker/StorageReportCSVDownloadWorker.java

```diff
@@ -57,20 +55,16 @@ public void run(ProgressCallback progressCallback, Message message) throws Excep
 		AsynchronousJobStatus status = asynchJobStatusManager.lookupJobStatus(message.getBody());
 		String fileName = "Job-"+status.getJobId();
 		File temp = null;
```

Comment:
Look at other file provider methods. Perhaps need to add a method that adds a Writer